userguide/system.rst

```diff
@@ -582,81 +582,81 @@ The configurable settings are summarized in
    |                                          |                                  |                                                                              |
    +------------------------------------------+----------------------------------+------------------------------------------------------------------------------+
-   | Remote Graphite Server hostname          | string                           | IP address or hostname of a remote server running                            |
-   |                                          |                                  | `Graphite <http://graphiteapp.org/>`__                                       |
+   | Remote Graphite Server hostname          | string                           | Enter the IP address or hostname of a remote server running                  |
+   |                                          |                                  | `Graphite <http://graphiteapp.org/>`__.                                      |
    |                                          |                                  |                                                                              |
    +------------------------------------------+----------------------------------+------------------------------------------------------------------------------+
-   | Use FQDN for logging                     | checkbox                         | when checked, include the Fully-Qualified Domain Name in logs to precisely   |
-   |                                          |                                  | identify systems with similar hostnames                                      |
+   | Use FQDN for logging                     | checkbox                         | Enable to include the Fully-Qualified Domain Name in logs to precisely       |
+   |                                          |                                  | identify systems with similar hostnames.                                     |
    +------------------------------------------+----------------------------------+------------------------------------------------------------------------------+
-   | ATA Security User                        | drop-down menu                   | specifies the command account used by legacy :ref:`Self-Encrypting Drives`;  |
-   |                                          |                                  | choices are *User* or *Master*                                               |
+   | ATA Security User                        | drop-down menu                   | Specifies the command account used by legacy :ref:`Self-Encrypting Drives`.  |
+   |                                          |                                  | Choices are *User* or *Master*.                                              |
    +------------------------------------------+----------------------------------+------------------------------------------------------------------------------+
```

Comment:
I think Specifies should be Specify. We're telling the user to specify the account.